File: Todo_Frontend/src/app/todos.ts
// To parse this data:
//
//   import { Convert } from "./file";
//
//   const employee = Convert.toEmployee(json);

export interface Employee {
    id:    number;
    name:  string;
    hours: number;
    title: string;
}

// Converts JSON strings to/from your types
export class ConvertEmployee {
    public static toEmployee(json: string): Employee[] {
        return JSON.parse(json);
    }

    public static employeeToJson(value: Employee[]): string {
        return JSON.stringify(value);
    }
}

// To parse this data:
//
//   import { Convert } from "./file";
//
//   const todos = Convert.toTodos(json);

export interface Todos {
    id:          number;
    name:        string;
    description: string;
    assignedTo:  number;
    isCompleted: boolean;
}

// Converts JSON strings to/from your types
export class ConvertTodos {
    public static toTodos(json: string): Todos[] {
        return JSON.parse(json);
    }

    public static todosToJson(value: Todos[]): string {
        return JSON.stringify(value);
    }
}

File: Todo_Backend/Todo_Backend/Models/Todos.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Todo_Backend.Models
{
    public partial class Employee
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(20)]
        public string? Name { get; set; }
        [Range(0, 40)]
        public double? Hours { get; set; }
        [MaxLength(40)]
        public string? Title { get; set; }

    }

    public partial class Todos
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(25)]
        public string? Name { get; set; }
        [MaxLength(100)]
        public string? Description { get; set; }
        [ForeignKey("Employee")]
        public int? AssignedTo { get; set; }
        public bool? IsCompleted { get; set; }
    }
}
